extensions/amp-link-rewriter/0.1/test/test-noscope-document.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Let's get this updated since its a new file.

Reply:
Done